src/layout/styles/dark.ts

```diff
@@ -38,6 +38,7 @@ const layoutDark: LayoutTheme = {
       siderColorInverted: cardColor,
       siderToggleButtonBorder: '1px solid transparent',
       siderToggleButtonColor: popoverColor,
+      siderToggleButtonTextColor: textColor2,
```

Comment:
```suggestion
      siderToggleButtonIconColor: textColor2,
```
再加一个 siderToggleButtonIconColorInverted，分开用